eZ/Bundle/EzPublishCoreBundle/Resources/views/content_fields.html.twig

```diff
@@ -357,55 +357,59 @@
 
     {% if hasContent and showMap %}
     {% set apiKey = ezpublish.configResolver.hasParameter('api_keys.google_maps') ? ezpublish.configResolver.getParameter('api_keys.google_maps'): "" %}
```

Comment:
This should be removed from here as well. Can you please add info that is config is deprecated now?